pkg/cloud/amazon/awscli/aws_cli.go

```diff
@@ -15,7 +15,7 @@ func NewAWSCli() awsClient {
 
 // ConnectToClusterWithAWSCLI will modify the kube-config file to add the provided cluster and change context to it
 func (awsClient) ConnectToClusterWithAWSCLI(clusterName string) error {
-	args := []string{"eks", "update-kubeconfig", "--name", clusterName}
```

Comment:
Which version of eks are you using? 

I am wondering if we should bump the version as well since is locked down here https://github.com/jenkins-x/jenkins-x-versions/blob/bdab4d2a65b04658d30707877a9072f05738ba9d/packages/eksctl.yml#L1.

Reply:
I think we have to. The version we install does not have the 'cluster' option yet.

Reply:
assuming '--name' still works in the latest version (but just produces a deprecation warning), it would be better to first update the version stream and once that has merged merge this pull request. @ccojocar wdyt?

Reply:
Yeah, this is why I am asking.

Reply:
@TheJokersThief, do you feel up for creating a pull request against the version repository as well?
@TheJokersThief, @ccojocar - any idea whether an upgrade could cause any incompatibilities or issues somewhere else?

Reply:
I am no sure if there are other flags which are deprecated. I think that it needs some validation before increasing the version.

Reply:
I'm not sure what other changes there are, this was an issue I encountered during `jx boot`. I had done `jx get eks` previously which had prompted me to download the dependencies (incl. eksctl), then during the `jx boot` resulted in the error in my PR description - it looks like it installed the right version but then I'm confused why my installer got the flag issue 🤔


```
$ jx get eks
Command "eks" is deprecated, it will be removed on 01-02-2020. We now highly recommend you use eksctl get clusters instead.
? Missing required dependencies, deselect to avoid auto installing: eksctl, aws-iam-authenticator
Installing eksctl
Downloading https://github.com/weaveworks/eksctl/releases/download/0.1.19/eksctl_Darwin_amd64.tar.gz to /Users/esmith/.jx/bin/eksctl.tar.gz...
Downloaded /Users/esmith/.jx/bin/eksctl.tar.gz
```

Reply:
Anyhow the versioned locked in our version stream is very old.  It seems that this flag was changed during 0.8.0 release https://github.com/weaveworks/eksctl/commit/15c8d2aec6dcbfba3da36488e02fd6d032cdabea#diff-13ea1eaae83a789bcd0a14f4291d715d.

Reply:
The name seems to be there but is only deprecated:

```
eksctl utils associate-iam-oidc-provider --name bla
Flag --name has been deprecated, use --cluster
[ℹ]  eksctl version 0.11.1
[ℹ]  using region us-east-1
[✖]  checking AWS STS access – cannot get role ARN for current session: ExpiredToken: The security token included in the request is expired
        status code: 403, request id: 53219469-2340-11ea-9ada-4b327e59e781
```

Reply:
This is latest 0.11.0.

Reply:
I would suggest that we bump first the version in our version stream to 0.11.0 and then merge this change if the BDD tests pass.

Reply:
I think that sounds like a good plan - do you want me to open a PR on the versions repo to bump the version?

Reply:
I increased the version in this PR https://github.com/jenkins-x/jenkins-x-versions/pull/725.

Reply:
Nevermind - you got it, thank you! :D

Reply:
Awesome, thank you :D